portfolio/src/main/webapp/index.html

```diff
@@ -150,7 +154,21 @@ <h2>Slide Show</h2>
 <div id="connect">
   <h2>Connect with me</h2>
   <p>
-    Follow me here
+    Follow me here<br>
+  <div class="container">
+    <div class="row mt-1">
+      <div class="col">
+        <a href="https://www.linkedin.com/in/prashantneupane" class="btn btn-dark" role="button" rel="noopener noreferrer">LinkedIn</a>
+      </div>
+      <div class="col">
+        <a href="https://www.instagram.com/shady_prashant/" class="btn btn-dark" role="button" rel="noopener noreferrer">Instagram</a>
```

Comment:
This line is a little long, consider wrapping attributes.

Reply:
Changes made